### DISCUSSION TITLE
LocalSend on CasaOS

### DISCUSSION THREAD

**Discussion by AAL994** [localsend/localsend#827] [Discussion (English, International)]
Hi folks! 👋 
Is it possible to install LocalSend on CasaOS? I don't know, but maybe using Docker?

**Comment by SeeDborg** [comment]
+1 on this, I would love to have Localsend on my CasaOS !

**Comment by Git-Vi** [comment]
I am really looking forward to start using the service in docker container

**Comment by V0LDY** [comment]
> Hi folks! 👋 Is it possible to install LocalSend on CasaOS? I don't know, but maybe using Docker?

CasaOS runs on top of another Linux system, why not just install LocalSend on that OS?

**Comment by SeeDborg** [comment]
I'd like to have the icon and be able to access it from CasaOS in my browser.

**Comment by Ast3risk-ops** [comment]
Localsend doesn't have a web GUI or a docker container as far as I know. It's meant for devices with graphical UIs to exchange files with one another. If you want a web version of this, host [Snapdrop](https://github.com/snapdrop/snapdrop) via CasaOS's custom container feature.

**Reply by AAL994** [reply]
Thank you for that suggestion; it’s exactly what I needed. 😊👍

**Comment by SeeDborg** [comment]
I already did it, though I would prefer to have Localsend...

**Reply by Ast3risk-ops** [reply]
How? It's not designed for web use, when Snapdrop is. It requires a sharing server, and can run on any device. There's little point for a web version.

**Comment by SeeDborg** [comment]
The point would be to use an app that I love and use everyday on all my devices.
I don't need a web version, a local version would be enough.

**Reply by Ast3risk-ops** [reply]
But you want it on CasaOS, which is a server software for hosting *web apps*.
